nemo_text_processing/g2p/modules.py

```diff
@@ -286,13 +289,20 @@ def __init__(
         """
         self.use_stresses = use_stresses
         self.set_graphemes_upper = set_graphemes_upper
+        self.use_chars = use_chars
+        self.phoneme_probability = phoneme_probability
+        self._rng = random.Random()
+
+        if not self.use_chars and self.phoneme_probability is not None:
+            logging.warning("use_chars was not set while using phoneme_probability.")
```

Comment:
The old default was to override and add chars in the conflicting; if we don't want that behavior I think this should error out instead.

Reply:
The interface is a bit confusing, but I cannot think of a downside to using the old default and automatically adding the graphemes from the dictionary when the probability is set. Will change.